autodispose-rxlifecycle/src/main/java/com/ubercab/autodispose/rxlifecycle/RxLifecycleInterop.java

```diff
@@ -40,6 +56,22 @@ private RxLifecycleInterop() {
     };
   }
 
+  /**
+   * Converter for  transforming {@link LifecycleProvider} to {@link ScopeProvider}.
```

Comment:
Nit: fix the double space after `for` in this and the one above

Reply:
Fixed - https://github.com/uber/AutoDispose/pull/118/commits/b373d422850c635444ef1c3ef48198544da7444e